_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -421,54 +352,33 @@ This [HTML document][] has a [keyboard shortcut][] using only a [printable chara
 </html>
 ```
 
```

Comment:
Need a failed example passing expectation 1 but failing expectation 2 (hmmm, I guess instrument in a modal and nothing identifying the modal or its opening button as a setting page).